GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -39,10 +41,22 @@ public final class EntryWidget: NSObject {
         self.environment = environment
         super.init()
 
-        environment.queuesMonitor.$state
-            .sink(receiveValue: handleQueuesMonitorUpdates(state:))
+        observeSecureUnreadMessageCount()
+
+        Publishers.CombineLatest(environment.queuesMonitor.$state, $unreadSecureMessageCount)
+            .sink(receiveValue: handleQueuesMonitorUpdates(state:unreadSecureMessagesCount:))
             .store(in: &cancellables)
     }
+
+    deinit {
+        if let unreadSecureMessageSubscriptionId {
+            environment.unsubscribeFromUpdates(unreadSecureMessageSubscriptionId) { [environment] error in
+                environment.log.prefixed(Self.self).warning(
+                    "Unsubscribing from unread secure messages count without configured SDK: \(error.localizedDescription)"
```

Comment:
This is newly introduce log message, so it has to go through [adding logs procedure](https://glia.atlassian.net/wiki/spaces/ENG/pages/3589734507/Logging+from+iOS+SDKs#Things-to-consider-for-newly-added-logs).

Reply:
You are right, I will mark this comment as resolved and sync with Android team, thanks!